cartoframes/context.py

```diff
@@ -1294,7 +1292,7 @@ def data_discovery(self, region, keywords=None, regex=None, time=None,
             except ValueError:
                 # TODO: make this work for general queries
                 # see if it's a table
-                self.sql_client.send(
+                self.batch_sql_client.create_and_wait_for_completion(
```

Comment:
Does batch SQL work with `EXPLAIN`?

Reply:
mm the point is that the result of the `sql_client` was not being stored anywhere so I assume the query is just to check if the table exists, so the `batch_sql_client` works for that purpose as well.

Reply:
It seems overkill to use the batch API (which might take a while to run due to scheduling). I'd use the normal SQL API if the change is easy.